pkg/accounts/mod.ts

```diff
@@ -106,16 +106,23 @@ accounts.doc('/accounts/doc.json', {
   },
 });
 
-accounts.openapi(CreateAccountRoute, async (c) => {
-  const { name, email, passphrase } = c.req.valid('json');
+export type AccountModuleHandlerType =
+  | typeof GetAccountHandler
+  | typeof CreateAccountHandler;
 
-  const res = await controller.createAccount(name, email, passphrase);
-  if (Result.isErr(res)) {
-    return c.json({ error: res[1].message }, { status: 400 });
-  }
+export const CreateAccountHandler = accounts.openapi(
```

Comment:
これらは export しなくてもよいのでは? `GetAccountHandler` も同じです
```suggestion
const CreateAccountHandler = accounts.openapi(
```

Reply:
`GetAccountHandler` の export は残ったままになっているので, まだ解決していないと思います. a80c61f は `const CreateAccountHandler` を除去したりする変更になっているようですが, このレビュー項目への対応を意図したものでしょうか?

Reply:
> GetAccountHandler の export は残ったままになっているので, まだ解決していないと思います. 

消すの忘れてました

> https://github.com/pulsate-dev/pulsate/commit/a80c61fc462e6df1211daa5d223b95c6e9674a00 は const CreateAccountHandler を除去したりする変更になっているようですが, このレビュー項目への対応を意図したものでしょうか?

CreateAccountHandlerは現状どこからも参照されていないので削除しました.